rock-paper-scissors/exercise-with-functions.js

```diff
@@ -0,0 +1,38 @@
+// Some practice using your Rock Paper Scissors code:
+// Move your code into a function called rockPaperScissors that only runs if you call rockPaperScissors()
+// Update your code so that the function rockPaperScissors takes in 2 answers (ie arguments) and outputs the winner.
+// calling function: rockPaperScissors('rock', 'paper')
+// should output: "Paper wins!"
+
+function sentenceCase(text) {
+  return text.charAt(0).toUpperCase() + text.substring(1);
+}
+
+function rockPaperScissors(playerOne, playerTwo) {
+  if (
+    playerOne !== "paper" &&
+    playerOne !== "rock" &&
+    playerOne !== "scissors"
+  ) {
+    return "Not a valid entry from Player One";
+  } else if (
+    playerTwo !== "paper" &&
+    playerTwo !== "rock" &&
+    playerTwo !== "scissors"
+  ) {
+    return "Not a valid entry from Player Two";
```

Comment:
Since this code checks the same thing but for different players you could pull it out into a function to avoid the repetition:
```
function isValidPlayer(player) {
  if (
    player !== "paper" &&
    player !== "rock" &&
    player !== "scissors"
  ) {
    return false
} else {
     return true
}
}
```

```suggestion
  if (!isValidPlayer(playerOne)) {
    return "Not a valid entry from Player One";
  } else if (!isValidPlayer(playerTwo)) {
    return "Not a valid entry from Player Two";
```